README.md

```diff
@@ -62,6 +62,27 @@ loggins.trackQNAEvent(context, userQuery, kbQuestion, kbAnswer, score);
 //You can see a working sample in [https://github.com/Microsoft/BotBuilder-CognitiveServices/tree/master/Node/samples/QnAMakerWithFunctionOverrides](https://github.com/Microsoft/BotBuilder-CognitiveServices/tree/master/Node/samples/QnAMakerWithFunctionOverrides)
 ```
 
+## Logging custom properties from session
+
+```js
+var instrumentation = require('botbuilder-instrumentation');
+
+// Setting up advanced instrumentation
+let logging = new instrumentation.BotFrameworkInstrumentation({ 
+  instrumentationKey: ["main insturmnation key","secondary instumentation key"],
+  sentimentKey: process.env.CG_SENTIMENT_KEY,
+  customFields: {
+    userData: [ "CUSTOM_PROPERTY" ]
```

Comment:
@morsh 
Like I wasn't aware of the structure of a `session`, I feel that the readme should explain how it works.
For example:
The Microsoft Bot Framework session object contains the following property bags: 
`[ 'userData', 'conversationData', 'privateConversationData', 'dialogData' ]`
and you can customize your log to include properties of those property-bags.
It can be done by specifying the name of the property bag, and an array of string which represents the name of the properties you would like to send to the telemetry.
For example:
```
customFields: {
   userData: [ "CUSTOM_PROPERTY" ]
  }
```